### PR TITLE
Added properties for bold/strong and large text class

### PR DESCRIPTION
**Pull request by aletail**
https://werkbotstudios.teamwork.com/app/tasks/31608869

### Testing Steps
- [x] Test with Erie Injury
- [x] Test updating the bold/strong tags
- [ ] Test updating the large text class
- [ ] Confirm these work as expected

### Issues/Concerns
- I think this is a rare case where we would alter these, but it doesn't hurt

### Git Flow
- **DO NOT** delete "release/\*" or "hotfix/\*" branches after merging a PR. These are used to publish the next release, and they are deleted automatically.
- "Squash and merge" is good on "feature/\*" into "develop"
- "Create a merge commit" is good on "release/\*" or "hotfix/\*" into "master"

### CODE REVIEW

**Review by tiller1010 — APPROVED** [review]
This works great! Just a question on the naming conventions used here for consistency.

File: sass/elements/typography/defaults/_text.scss [hunk 1]
```diff
@@ -1,5 +1,16 @@
+// Default text Properties
+$default-text-properties:(
+    boldStrongWeight: 700,
+    largeTextFontSize: 1.5em,
+);
+// If $default-text-properties is set, lets merge into our defaults
+$text-properties: $default-text-properties !default;
```
[the remark this answers is not in the document]
Comment (aletail) [reply sass/elements/typography/defaults/_text.scss:7]: Ya, I struggled with this when doing it...it doesn't really fall inline with element or component

[the remark this answers is not in the document]
Comment (aletail) [reply sass/elements/typography/defaults/_text.scss]: I ended up going with `element` here:
- at first I thought about moving this file inside the "elements" folder, but decided to leave it as is
- I also thought about using plural, but thought that may lead to some user errors like forgetting the "s"
- So ended up just going with `element`

Let me know your thoughts

[the remark this answers is not in the document]
Comment (tiller1010) [reply sass/elements/typography/defaults/_text.scss]: I think "element" makes sense in this particular case. I think the original idea behind putting these elements into one "text" file was to reduce the amount of `$property` maps and files. We would need a lot more of each if we did these text elements individually. `$element-italic-properties` may be a bit overkill for an element with a single style rule.

**Review by tiller1010 — APPROVED** [review]
This looks good!

**Pull request merged**